src/test/java/com/celements/cells/CellRenderStrategyTest.java

```diff
@@ -199,6 +199,7 @@ public void test_startRenderCell() throws Exception {
     BaseObject cellObj = addCellObj(doc);
     String cssClasses = "classes two";
     String idname = "myDivId";
+    String cellRef = "Skin.MasterCell";
```

Comment:
please call it according content
```suggestion
    String cellFN = "Skin.MasterCell";
```